components/markets/ScalarPriceRange.tsx

```diff
@@ -91,42 +92,49 @@ const ScalarPriceRange = observer(
               <span className="mb-2.5 text-sm text-red">{upper}</span>
             </div>
           </div>
-          {showShortAndLongPrices && (
+          {/* TODO: check if this can be removed */}
```

Comment:
Can this be removed (also line 130?)? Seems like it's no longer used but not exactly sure what it does

Reply:
it used to show horizontal lines around the actual current prediction in the middle when `shortPrice + longPrice` is < 0.97, but now it seems this sum is always very near 1 so it doesn't show. Just leave it commented out for now.